src/entt/entity/organizer.hpp

```diff
@@ -263,11 +265,27 @@ class basic_organizer final {
         }
 
         /**
-         * @brief Returns the list of nodes reachable from a given vertex.
-         * @return The list of nodes reachable from the vertex.
+         * @brief Returns the list of predecessors for a given vertex.
+         * @return The list of predecessors' indices for the vertex.
+         */
+        const std::vector<std::size_t> &predecessors() const noexcept {
+            return predecessors_vec;
+        }
+
+        /**
+         * @brief Returns the list of successors for a given vertex.
+         * @return The list of successors' indices for the vertex.
+         */
+        const std::vector<std::size_t> &successors() const noexcept {
+            return successors_vec;
+        }
+
+        /**
+         * @brief Returns the list of successors for a given vertex.
+         * @return The list of successors' indices for the vertex.
          */
         const std::vector<std::size_t> &children() const noexcept {
```

Comment:
Keep this one here is fine but I'd add a `[[deprecated("use ::successors")]]` too.